Dockerfile

```diff
@@ -3,14 +3,18 @@ ENV DEBIAN_FRONTEND noninteractive
 
 RUN apt-get -qq -y update \
     && apt-get -qq -y install python3-pip \
+    pkg-config libicu-dev \
     && apt-get -qq -y autoremove \
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
 
+RUN pip3 install --no-binary=:pyicu: pyicu
```

Comment:
Can you leave a brief comment as to why we need that special flag? Otherwise could we not just put this into `requirements.txt`?

Reply:
The command is chosen based on the [PyICU docs](https://pypi.org/project/PyICU/), in the Debian section.